markdown-link-check.json

```diff
@@ -1,13 +1,21 @@
 {
   "ignorePatterns": [
     {
-      "pattern": "^https://github.com"
+      "description": "Ignore root relative links to images which markdown-link-check is unable to verify",
+      "pattern": "^/"
     },
     {
-      "pattern": "^https://acme.example.com"
+      "description": "Ignore in-page anchor links defined using <a id=''> syntax,. See https://github.com/tcort/markdown-link-check/issues/225",
+      "pattern": "^#"
     },
     {
-      "pattern": "^https://developers.cloudflare.com"
+      "description": "Ignore external links which may fail due to network flakiness and rate limits on foreign servers",
+      "pattern": "^https?://"
+    },
```

Comment:
For example
* https://github.com/cert-manager/website/actions/runs/3958447700/jobs/6780031828
```
FILE: content/docs/tutorials/getting-started-aks-letsencrypt/README.md
  [✖] https://www.cloudflare.com/en-gb/learning/dns/glossary/what-is-a-domain-name-registrar/
  [✖] https://www.cloudflare.com/en-gb/learning/dns/dns-server-types/
  [✖] clusterissuer-selfsigned.yaml
  [✖] certificate.yaml

  ERROR: 11 dead links found!
  [✖] deployment.yaml
  [✖] service.yaml
  [✖] https://www.cloudflare.com/en-gb/learning/dns/dns-records/dns-cname-record/
  [✖] ../../configuration/acme/dns01/azuredns/README.md
  [✖] values.yaml
  [✖] clusterissuer-lets-encrypt-staging.yaml
  [✖] clusterissuer-lets-encrypt-production.yaml

  34 links checked.
  [✖] https://www.cloudflare.com/en-gb/learning/dns/glossary/what-is-a-domain-name-registrar/ → Status: 403
  [✖] https://www.cloudflare.com/en-gb/learning/dns/dns-server-types/ → Status: 403
  [✖] clusterissuer-selfsigned.yaml → Status: 400
```